monocle/gerrit/review.py

```diff
@@ -116,6 +124,9 @@ def get(self, updated_since, change=None):
                     break
             else:
                 break
+        if self.prefix:
+            for review in reviews:
+                review["project"] = self.prefix + review["project"]
```

Comment:
Line 165 and 167 the computation for repository_prefix and repository_shortname won't work as expected for prefix such as "rdo/" as we split on "/".